src/components/Popup/Popup.scss

```diff
@@ -75,15 +75,17 @@ $transition-distance: 10px;
     }
 
     &[data-popper-placement*='left'] #{$block}__arrow {
-        inset-inline-end: -$arrow-offset;
+        // stylelint-disable-next-line csstools/use-logical
```

Comment:
Should we add todo here?

Reply:
No, popper.js uses physical properties, it doesn't support RTL, we should migrate to Floating UI, which does